manual/source/classes/contributed_definitions/ellipsometry-structure.rst

```diff
@@ -0,0 +1,179 @@
+       
+       Use two or more NXbeam_paths to describe the beam paths after the beam
+       splitter. In the dependency chain of the new beam paths, the first elements
+       each point to this beam splitter, as this is the previous element.
+
+    :ref:`NXfiber`
+       An optical fiber, e.g. glass fiber.
+
+    :ref:`NXlens_opt`
+       Description of an optical lens.
+       
+    :ref:`NXpolarizer_opt`
+       A spin polarizer.
+
+    :ref:`NXwaveplate`
+       A waveplate or retarder.
+
+    :ref:`NXenvironment`
+       Specify external parameters that have influenced the sample,
+       such as the surrounding medium, and varied parameters e.g.
+       temperature, pressure, pH value, optical excitation etc.
+
+
+
+.. _DispersiveMaterial:
+
+Dispersive Material
+###################
+
+A dispersive material is a description for the optical dispersion of materials.
+This description may be used to store optical model data from an ellipsometric analysis 
+(or any other technique) or to build a database of optical constants for optical properties of materials.
+
+Application Definition
+----------------------
+
+    :ref:`NXdispersive_material`:
+       An application definition to describe the dispersive properties of a material.
+       The material may be isotropic, uniaxial or biaxial. Hence, it may contain up
+       to three dispersive functions or tables.
+
+
+
+Base Classes
+------------
+
+There is a set of base classes for describing a dispersion.
+
+    :ref:`NXdispersion`
+       This is an umbrella base class for a group of dispersion functions to describe the material.
+       For a simple dispersion it may contain only on NXdispersion_function or NXdispersion_table entry.
+       If it contains multiple entries the actual dispersion is the sum of all dispersion functions and tables.
+       This allows for, e.g. splitting real and imaginary parts and describing them seperately or
+       adding a dielectric background (e.g. Sellmeier model) to an oscillator model (e.g. Lorentz).
+              
+    :ref:`NXdispersion_function`
+       This dispersion is described by a function and its single and repeated parameter values.
+       It follows a formula of the form ``eps = eps_inf + sum[A * lambda ** 2 / (lambda ** 2 - B ** 2)]`` 
+       (Sellmeier formula). See the formula grammar below for an ebnf grammar for this form.
+
+    :ref:`NXdispersion_single_parameter`
+       This denotes a parameter which is used outside the summed part of a dispersion function,
+       e.g. ``eps_inf`` in the formula example above.
+
+    :ref:`NXdispersion_repeated_parameter`
+       This denotes arrays of repeated parameters which are used to build a sum of parameter values, e.g.
+       ``A`` and ``B`` are repeated parameters in the formula above.
+       
+    :ref:`NXdispersion_table`
+       This describes a tabular dispersion where the dielectric function is an array versus wavelength or energy.
```

Comment:
What's the dielectric function? Is `eps` above the permittivity?

Reply:
It basically is the permittivity, i.e., complex `eps` which is the square of the complex refactive index. If this is unclear/jargony we can just rename this to permittivity.

Reply:
The British terminology is dielectric constant (the real part of the relative permittivity). It's best to limit the number of new terms/jargon to the minimal.